Packs/Indeni/Integrations/Indeni/Indeni.py

```diff
@@ -0,0 +1,376 @@
+            devices.append({
+                'DeviceName': device_name,
+                'DeviceId': alert.get('device_id'),
+                'Items': items
+            }
+            )
+
+    content = {
+        'Indeni.AffectedDevices(val.AlertType == obj.AlertType)': {
+            'AlertType': alert_type,
+            'Device': devices
+        }
+    }
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': content,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(f'Devices Experiencing Alert {alert_type}', devices, removeNull=True),
+        'EntryContext': content
+    })
+
+
+def post_note(base_url):
+    alert_id = demisto.args().get('alert_id')
+    alert_note = demisto.args().get('note')
+
+    endpoint_url = 'issues/' + alert_id + '/notes'
+    body = {
+        'text': alert_note
+    }
+    http_request('POST', base_url + endpoint_url, data=body)
+
+
+def get_notes(base_url):
+    alert_id = demisto.args().get('alert_id')
+    endpoint_url = 'issues/' + alert_id + '/notes'
+    response = http_request('GET', base_url + endpoint_url)
+    notes = []
+    for note in response:
+        notes.append({
+            'note': note.get('text'),
+            'timestamp': timestamp_to_datestring(note.get('timestamp'))
+        })
+    content = {
+        'Indeni.AlertInfo(val.AlertID == obj.AlertID).Note': notes
+    }
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': content,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('Issue Notes', notes, removeNull=True),
+        'EntryContext': content
+    })
+
+
+def archive_issue(base_url):
+    alert_id = demisto.args().get('alert_id')
+    endpoint_url = 'issues/' + alert_id
+    body = {
+        'archived': True
+    }
+    http_request('PATCH', base_url + endpoint_url, data=body)
+
+
+def unarchive_issue(base_url):
+    alert_id = demisto.args().get('alert_id')
+    endpoint_url = 'issues/' + alert_id
+    body = {
+        'archived': False
+    }
+    http_request('PATCH', base_url + endpoint_url, data=body)
+
+
+# Indeni: CRITICAL: 0, ERROR: 1, WARN: 2, INFO: 3
+def issue_severity_to_issue_level(issue_severity):
+    if issue_severity == 'CRITICAL':
+        return 0
+    elif issue_severity == 'ERROR':
+        return 1
+    elif issue_severity == 'WARN':
+        return 2
+    elif issue_severity == 'INFO':
+        return 3
+    else:
+        return -1
+
+
+def fetch_incidents(base_url):
+    last_run = demisto.getLastRun()
+    # Get the last fetch time, if exists
+    alert_id_index = last_run.get('alert_id', 0)
+    only_pan_cve_issues = demisto.params().get('onlyPullPanCveIssues', False)
+    max_pull_size = demisto.params().get('maxPullSize', 20)
+    lowest_issue_severity = demisto.params().get('issueSeverity', [])
```

Comment:
put here tha value you would have put in the default:
`lowest_issue_severity = demisto.params().get('issueSeverity', 'INFO')`

Reply:
Done